### DISCUSSION TITLE
General Availability

### DISCUSSION THREAD

**Discussion by lando213** [awslabs/aws-sdk-swift#1071] [Q&A] [answered]
Is there any GA target date for the SWIFT API?

**Comment by jbelkins** [comment, marked as answer]
Sorry, we don't have a target date.  We will update customers when the Swift SDK is generally available.

It is fair to say that the AWS SDK for Swift will not reach GA in calendar year 2023 though.  I hope that provides a bit of clarity for your planning purposes.

If you need a SDK for Apple platforms that is generally available now, please look at https://github.com/aws-amplify/aws-sdk-ios .

**Reply by lando213** [reply]
Thank!! Much appreciated.

**Reply by lando213** [reply]
I went to aws-ios-sdk but it says V2 is built on top of and provides only access to the SWIFT sdk. So, should I use V1 or am I stuck waiting for GA?

**Comment by jbelkins** [comment]
We're happy to announce that Swift SDK is now generally available.

The latest SDK release is always available on this Github repo.
